client/js/table-view.js

```diff
@@ -18,6 +18,7 @@ class TableView {
     this.headerRowEl = document.querySelector('THEAD TR');
     this.sheetBodyEl = document.querySelector('TBODY');
     this.formulaBarEl = document.querySelector('#formula-bar');
+    this.sumRowEl = document.querySelector('TFOOT TR')
```

Comment:
"sumRowEl" is not a bad name, but what you are doing here is very similar to "headerRowEl", only this time with a footer element. Keep your naming conventions consistent throughout the code. It would be a little closer to the patterns to call this "footerRowEl", for instance.